src/components/ItemEditorPage/RightPanel/RightPanel.tsx

```diff
@@ -309,8 +354,12 @@ export default class RightPanel extends React.PureComponent<Props, State> {
               let hides: WearableCategory[] = []
 
               if (data) {
```

Comment:
I believe that if we add `&& `isEmoteData(data)` here, the `as WearableData` underneath won't be needed.

Reply:
@juanmahidalgo thanks for the review! 

Updated!